build/pmm-server-docker/Dockerfile

```diff
@@ -1,5 +1,19 @@
 FROM centos:7
 
+ARG VERSION
+ARG BUILD_DATE
+
+LABEL org.label-schema.build-date ${BUILD_DATE}
+LABEL org.label-schema.license AGPL-3.0
+LABEL org.label-schema.name Percona Monitoring and Management
+LABEL org.label-schema.vendor Percona
+LABEL org.label-schema.version ${VERSION}
+LABEL org.opencontainers.image.created ${BUILD_DATE}
+LABEL org.opencontainers.image.licenses AGPL-3.0
+LABEL org.opencontainers.image.title Percona Monitoring and Management
+LABEL org.opencontainers.image.vendor Percona
+LABEL org.opencontainers.image.version ${VERSION}
+
```

Comment:
oops, looks like `"org.label-schema.schema-version" is set to "1.0" by the upstream image. Maybe we can set to `2.x` meaning that it has to do with PMM v2?

Reply:
`schema-version` is version of schema: http://label-schema.org/rc1/
It's just standard version like http 2.0 :)